src/ontology/cl-edit.owl

```diff
@@ -8936,11 +8936,17 @@ SubClassOf(obo:CL_0000641 obo:CL_0000163)
 
 # Class: obo:CL_0000642 (folliculostellate cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb") obo:IAO_0000115 obo:CL_0000642 "A supportive cell of the vertebrate pituitary that provides macromolecular transport and secretes hormones.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:10495875") Annotation(oboInOwl:hasDbXref "PMID:15961560") Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") obo:IAO_0000115 obo:CL_0000642 "An agranular supporting cell of the anterior pituitary (adenohypophysis) that is characterized by a star-like morphology and ability to form follicles. Folliculostellate cells communicate with each other and with endocrine cells via gap junctions.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") oboInOwl:hasExactSynonym obo:CL_0000642 "folliculo-stellate cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34734454") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_0000642 "FS cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_0000642 "FSC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34734454") oboInOwl:hasRelatedSynonym obo:CL_0000642 "Sox2-positive stem cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18287078") Annotation(oboInOwl:hasDbXref "PMID:31620083") Annotation(oboInOwl:hasDbXref "PMID:34734454") Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") rdfs:comment obo:CL_0000642 "Folliculostellate cells are a heterogeneous group of cells rather than a single cell type, with different morphology and gene expression profiles. Due this heterogeneity, it is unclear if different properties that have been attributed to folliculostellate cells (pluripotency, proliferative, phagocytic) are common in all folliculostellate cells or particular to a subset.")
```

Comment:
Some minor grammar edits:

Folliculostellate cells are a heterogeneous group of cells rather than a single cell type, with different morphology and gene expression profiles. Due **to** this heterogeneity, it is unclear if different properties that have been attributed to folliculostellate cells (**pluripotent**, proliferative, phagocytic) are common in all folliculostellate cells or **limited to particular subsets**.
```